### ISSUE TITLE
see no difference in time and quality with cublas on\off, files are the same with flash_attn

### ISSUE THREAD

**Issue by ptits** [kijai/ComfyUI-MochiWrapper#37]
https://github.com/user-attachments/assets/3ec90251-32f0-4a4e-ad34-0611e6982472

https://github.com/user-attachments/assets/b842af65-c1ec-43d8-b70f-57ec7321bf8b

**Comment by ptits** [comment]
> Did you install it? https://github.com/aredden/torch-cublas-hgemm

did it now

pip list |grep cub
cublas_ops                0.0.5
nvidia-cublas-cu12        12.4.5.8

made tests again

still no differece


got prompt
Initializing model...
Loading model state_dict from /home/jovyan/ComfyUI/models/diffusion_models/mochi/mochi_preview_dit_bf16.safetensors...
Using accelerate to load and assign model weights to device...
Using sigma_schedule: [1.0, 0.999, 0.998, 0.997, 0.996, 0.995, 0.994, 0.993, 0.992, 0.991, 0.99, 0.989, 0.988, 0.987, 0.986, 0.985, 0.984, 0.983, 0.982, 0.981, 0.98, 0.979, 0.978, 0.977, 0.976, 0.9749999999999999, 0.9724799999999999, 0.9669199999999999, 0.95832, 0.9466799999999996, 0.9319999999999998, 0.9142799999999996, 0.8935199999999998, 0.8697199999999999, 0.8428799999999997, 0.8129999999999998, 0.7800799999999996, 0.7441199999999998, 0.7051200000000001, 0.6630799999999996, 0.618, 0.5698799999999996, 0.5187199999999997, 0.46451999999999993, 0.40727999999999975, 0.34699999999999964, 0.2836799999999996, 0.21731999999999962, 0.14791999999999927, 0.07547999999999944, 0.0]
Using cfg schedule: [4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5]
Processing Samples: 100%|██████████████████████████████████████████████████████████████| 50/50 [01:35<00:00,  1.92s/it]
samples shape: torch.Size([1, 12, 9, 60, 106])
Frame batch size is larger than the number of samples, setting to 9
Processing rows: 100%|███████████████████████████████████████████████████████████████████| 4/4 [00:02<00:00,  1.52it/s]
Blending rows: 100%|█████████████████████████████████████████████████████████████████████| 4/4 [00:00<00:00, 32.09it/s]
[VideoHelperSuite] - WARNING - Output images were not of valid resolution and have had padding applied                 
Prompt executed in 120.63 seconds

got prompt
Initializing model...
Loading model state_dict from /home/jovyan/ComfyUI/models/diffusion_models/mochi/mochi_preview_dit_bf16.safetensors...
Using accelerate to load and assign model weights to device...
Using sigma_schedule: [1.0, 0.999, 0.998, 0.997, 0.996, 0.995, 0.994, 0.993, 0.992, 0.991, 0.99, 0.989, 0.988, 0.987, 0.986, 0.985, 0.984, 0.983, 0.982, 0.981, 0.98, 0.979, 0.978, 0.977, 0.976, 0.9749999999999999, 0.9724799999999999, 0.9669199999999999, 0.95832, 0.9466799999999996, 0.9319999999999998, 0.9142799999999996, 0.8935199999999998, 0.8697199999999999, 0.8428799999999997, 0.8129999999999998, 0.7800799999999996, 0.7441199999999998, 0.7051200000000001, 0.6630799999999996, 0.618, 0.5698799999999996, 0.5187199999999997, 0.46451999999999993, 0.40727999999999975, 0.34699999999999964, 0.2836799999999996, 0.21731999999999962, 0.14791999999999927, 0.07547999999999944, 0.0]
Using cfg schedule: [4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5, 4.5]
Processing Samples: 100%|██████████████████████████████████████████████████████████████| 50/50 [01:35<00:00,  1.92s/it]
samples shape: torch.Size([1, 12, 9, 60, 106])
Frame batch size is larger than the number of samples, setting to 9
Processing rows: 100%|███████████████████████████████████████████████████████████████████| 4/4 [00:02<00:00,  1.52it/s]
Blending rows: 100%|█████████████████████████████████████████████████████████████████████| 4/4 [00:00<00:00, 32.06it/s]
[VideoHelperSuite] - WARNING - Output images were not of valid resolution and have had padding applied                 
Prompt executed in 121.70 seconds

**Comment by ptits** [comment]
by the way if i switch cublas_ops on\off model is not reloaded (no "Initializing model..." above)

may be because i use Loader, and not (Down)loader?

![shot_241028_014729](https://github.com/user-attachments/assets/82126fb1-8e5e-4aac-8188-b48c5fcfc097)

**Comment by ptits** [comment]
> Cublas_ops is only used with GGUF models.

Thanks, now I see

Loading GGUF model state_dict...
Using cublas_ops

I see almost no difference in generation time (1.16s/it vs1.17s/it)

but results are really different with cublas_ops on\off


https://github.com/user-attachments/assets/4422461a-9769-4459-8320-04c35b54f65d
